modules/widgets/simple-payments/customizer.js

```diff
@@ -70,6 +73,31 @@
 		widgetForm.find( '.jetpack-simple-payments-delete-product' ).on( 'click', deleteProduct( widgetForm ) );
 	}
 
+	function syncProductLists() {
+		var request = wp.ajax.post( 'customize-jetpack-simple-payments-buttons-get', {
+			'customize-jetpack-simple-payments-nonce': api.settings.nonce[ 'customize-jetpack-simple-payments' ],
+			'customize_changeset_uuid': api.settings.changeset.uuid
+		} );
+
+		request.done( function( data ) {
+			var selectedProduct = 0;
+
+			$( document ).find( 'select.jetpack-simple-payments-products' ).each( function( index, select ) {
+				var $select = $( select );
+				selectedProduct = $select.val();
+
+				$select
+					.find( 'option' )
+					.remove()
+					.end() //cancels the find operation and returns the original <select /> element.
```

Comment:
Let's make this into multiple statements, if folks are finding this difficult to read. eg

```javascript
$select.find( 'option' ).remove();
$select.append( addOptions );
$select.val( selectedProduct );
```

Reply:
updated on 8e8f21b55a67a7e0d97b8dcd51a2e1c1c0ac7fca.